registration/models.py

```diff
@@ -274,16 +274,19 @@ def delete_expired_users(self):
             models.Q(user__is_active=False) | models.Q(user=None),
             activated=False,
         )
+        deleted_count = 0
         for profile in profiles:
             try:
                 if profile.activation_key_expired():
                     user = profile.user
                     logger.warning('Deleting expired Registration profile {} and user {}.'.format(profile, user))
                     profile.delete()
                     user.delete()
+                    deleted_count += 1
             except UserModel().DoesNotExist:
                 logger.warning('Deleting expired Registration profile {}'.format(profile))
                 profile.delete()
```

Comment:
i think you should increment the count here as well since it results in a profile being deleted

Reply:
I missed it I will send pr again.